### ISSUE TITLE
Нету поддержки от stm32

### ISSUE THREAD

**Issue by zDimaBY** [GyverLibs/mString#3]
https://github.com/rogerclarkmelbourne/Arduino_STM32

**Comment by GyverLibs** [comment]
Видимо трюки местного компилятора

**Comment by zDimaBY** [comment]
Да, там что-то с #include stdlib.h , нет функций ltoa, utoa .. и другого. Какие то зависимости.

**Comment by GyverLibs** [comment]
это стандартные функции gcc, возможно их нужно как то отдельно подключать
